### ISSUE TITLE
deploy to google app engine

### ISSUE THREAD

**Issue by lheagy** [ubcgif/gpg#86]
- [x] set up google app
- [x] autodeploy from travis
    - [x] set up encrypted credentials
- [x] ensure old links re-direct properly
- [x] add back "edit on github" button
- [x] tear down rtd deploy
- [x] put on gpg.geosci.xyz subdomain

**Comment by lheagy** [comment]
v0 is deploying to http://gpggeosci.appspot.com/

**Comment by fourndo** [comment]
you're the best for deploying so quickly.
I managed to push my animations on the current site :| 
Looks like we are ok under 5 Mb.
So no rush to migrate, on my end anyway.

**Comment by lheagy** [comment]
also need to address the gae upload errors
```
2016-09-10 15:17:40,805 ERROR appcfg.py:1867 Invalid character in filename: GPG Edits to make, April 2015.docx 
2016-09-10 15:17:41,222 ERROR appcfg.py:1867 Invalid character in filename: content/magnetics/images/applet-bg,gif.gif 
2016-09-10 15:17:41,331 ERROR appcfg.py:1867 Invalid character in filename: content/magnetics/images/lawn-post,gif.gif 
03:17 PM Scanned 500 files.
03:17 PM Scanned 1000 files.
2016-09-10 15:17:42,295 ERROR appcfg.py:1867 Invalid character in filename: _build/html/_images/applet-bg,gif.gif 
2016-09-10 15:17:42,834 ERROR appcfg.py:1867 Invalid character in filename: _build/html/_images/lawn-post,gif.gif 
```

**Comment by lheagy** [comment]
closed with #92. This means we are no longer restricted by size and can use python computations to generate figures :)
